#
# Automatically load localauth_gsscred plugin.
#
# To preserve long standing Solaris Kerberos behavior with respect to
# gsscred we always load the localauth_gsscred plugin delivered from ON gate.
#
# This is a Solaris specific patch, it is not meant for upstream contribution.
# Patch source: in-house
#
--- a/src/lib/krb5/os/localauth.c
+++ b/src/lib/krb5/os/localauth.c
@@ -133,6 +133,10 @@ get_modules(krb5_context context, krb5_plugin_initvt_fn **modules_out)
     if (ret)
         return ret;
 
+    ret = k5_plugin_register_dyn(context, intf, "gsscred", "localauth");
+    if (ret)
+        return ret;
+
     ret = k5_plugin_load_all(context, intf, modules_out);
     if (ret)
         return ret;
